tests/zfs-tests/tests/functional/cli_root/zpool_split/Makefile.am

```diff
@@ -9,4 +9,6 @@ dist_pkgdata_SCRIPTS = \
 	zpool_split_devices.ksh \
 	zpool_split_encryption.ksh \
 	zpool_split_props.ksh \
-	zpool_split_vdevs.ksh
+	zpool_split_vdevs.ksh \
+	zpool_split_resilver1.ksh \
```

Comment:
You need to add these to the linux.run file ("tests/runfiles/linux.run") otherwise they will not be picked up by the runner script.